app/src/test/kotlin/fr/free/nrw/commons/media/MediaDetailPagerFragmentUnitTests.kt

```diff
@@ -84,6 +88,7 @@ class MediaDetailPagerFragmentUnitTests {
         Fresco.initialize(context)
 
         val activity = Robolectric.buildActivity(SearchActivity::class.java).create().get()
+        binding = FragmentMediaDetailPagerBinding.inflate(activity.layoutInflater)
 
         fragment = MediaDetailPagerFragment.newInstance(false, true)
```

Comment:
Looks like this line is redundant since the variable is reinitialized in the next line anyhow.